login-workflow/src/new-architecure/components/workflow/WorkflowCardActions.tsx

```diff
@@ -0,0 +1,100 @@
+import React from 'react';
+import CardActions from '@mui/material/CardActions';
+import { Theme, useTheme } from '@mui/material/styles';
+import MobileStepper from '@mui/material/MobileStepper';
+import Button from '@mui/material/Button';
+import { useLanguageLocale } from '../../../auth-shared';
+
+type DataObject = { [key: string]: any };
+
+export type WorkflowCardActionsProps = {
+    canGoNext?: boolean | ((data: DataObject) => boolean);
+    canGoPrevious?: boolean | ((data: DataObject) => boolean);
+    showPrevious?: boolean;
+    showNext?: boolean;
+    previousLabel?: string;
+    nextLabel?: string;
+    onPrevious?: (data?: any) => void;
+    onNext?: (data?: any) => void;
+    currentStep?: number;
+    totalSteps?: number;
+    fullWidthButton?: boolean;
+    data?: DataObject;
+};
+
+const StepperDotStyles = (theme: Theme): any => ({
+    m: `0px ${theme.spacing(0.5)}`,
+});
+
+export const WorkflowCardActions: React.FC<WorkflowCardActionsProps> = (props) => {
```

Comment:
This should extend from CardActionsProps and spread the remaining to the element